internal/model/model.go

```diff
@@ -10,11 +10,12 @@ import (
 )
 
 type Model struct {
-	Slides   []string
-	Page     int
-	Author   string
-	Date     string
-	viewport viewport.Model
+	Slides         []string
+	Page           int
+	Author         string
+	Date           string
+	RendererOption glamour.TermRendererOption
```

Comment:
I think we can name this `Theme`, since thats a more descriptive name for the `RendererOption`